src/main/java/com/courtcanva/ccfranchise/services/MailingService.java

```diff
@@ -0,0 +1,30 @@
+package com.courtcanva.ccfranchise.services;
+
+import com.amazonaws.util.Base64;
+import com.courtcanva.ccfranchise.configs.MailingConfig;
+import com.courtcanva.ccfranchise.exceptions.MailingClientException;
+import com.courtcanva.ccfranchise.utils.MailingClient;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class MailingService {
+
+    private final MailingClient mailingClient;
+    private final MailingConfig mailingConfig;
+
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    public void sendVerificationEmail(String to, String verificationToken) throws MailingClientException {
+        String from = mailingConfig.getSender();
+        String subject = "Verify your CourtCanva credential!";
+        String verificationUrl = mailingConfig.getClientSideBaseUrl() + "/staff/verify-email?token=" + verificationToken + "&i=" + Base64.encodeAsString(to.getBytes());
+        String content = "<a target=\"_blank\" href=\"" + verificationUrl + "\">" + verificationUrl + "</a>";
+
```

Comment:
avoid hardcode strings in code

Reply:
Done, I've refactored `line 24: subject` and `line 25: /staff/verify-email` into their own static string variable at the top of the class.

As for `line 26: content`, will we be creating a template for verification emails on SES? if we do, perhaps we can keep `content` as it is until it's refactored to use an email template instead.